android/app/src/main/java/com/tourneynizer/tourneynizer/model/TournamentDef.java

```diff
@@ -0,0 +1,119 @@
+package com.tourneynizer.tourneynizer.model;
+
+import android.graphics.Bitmap;
+import android.location.Address;
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+import com.google.android.gms.location.places.Place;
+
+import java.sql.Time;
+import java.util.Locale;
+
+/**
+ * Created by ryanl on 2/3/2018.
+ */
+
+public class TournamentDef {

```

Comment:
Whats the difference between TournamentDef and Tournament model?

Reply:
TournamentDef is what I use to store data for when a User is trying to create a tournament. I decided to use this rather than a Tournament because I wanted to preserve the immutability and no null fields in my tournament class